doc/Translation.md

```diff
@@ -0,0 +1,85 @@
+# Translation
```

Comment:
Maybe expand the header? Something like: "Translating Privacy Badger"

Reply:
Yes, good idea